rest_api/controller/file_upload.py

```diff
@@ -19,13 +20,9 @@
 router = APIRouter()
 
 try:
-    pipeline_config = Pipeline._read_pipeline_config_from_yaml(Path(PIPELINE_YAML_PATH))
-    pipeline_definition = Pipeline._get_pipeline_definition(
-        pipeline_config=pipeline_config, pipeline_name=INDEXING_PIPELINE_NAME
-    )
-    definitions = Pipeline._get_component_definitions(
-        pipeline_config=pipeline_config, overwrite_with_env_variables=True
-    )
+    pipeline_config = read_pipeline_config_from_yaml(Path(PIPELINE_YAML_PATH))
+    pipeline_definition = get_pipeline_definition(pipeline_config=pipeline_config, pipeline_name=INDEXING_PIPELINE_NAME)
+    definitions = get_component_definitions(pipeline_config=pipeline_config, overwrite_with_env_variables=True)
```

Comment:
I'd call it `node_definitions` just to contrast with `pipeline_definitions`

Reply:
I'm gonna call it `component_definitions`

Reply:
done